pythonforandroid/bootstraps/library/build/templates/Service.tmpl.java

```diff
@@ -0,0 +1,113 @@
+package {{ args.package }};
```

Comment:
Are you sure this file is needed?

Reply:
This file contains code for setting service properties (like its entry point) and code for unpacking python bundle. I'm not sure that it is needed, but I don't know how to do these things without it.